### ISSUE TITLE
Relative image loading doesn't work if not nested in .tmx file in directory hierarchy.

### ISSUE THREAD

**Issue by radicaled** [vnen/godot-tiled-importer#38]
Due to some work in progress I had a directory structure like this:

```
+ game-root
   - campaigns
     - test_campaign              # Original test files
       - test_campaign.tmx
       - test_campaign.tsx
       - test_campaign.png
     - assets                           # "Production" assets, mid-organization of directory structure
       - test_campaign
          - new_campaign_assets.tsx    # Used by test_campaign.tmx
          - new_campaign_assets.png   # Used by new_campaign_assets.tsx
```

However, the importer can't find "new_campaign_assets.png" because it's checking `/game-root/campaigns/test_campaign`. I think this is because options.basedir is, and when a .tsx and its .png file are side-by-side there's no absolute path involved, just a relative one.

**Comment by jonathlela** [comment]
That's funny, I think I ran on the same kind of problem and just submitted a pull request to fix that.

**Comment by radicaled** [comment]
I thought about getting elbow deep into it after lunch but you've beaten me to it by far. :)

**Comment by vnen** [comment]
I imagine this was fixed by #39.
